config/confighttp/confighttp.go

```diff
@@ -46,6 +46,9 @@ type HTTPClientSettings struct {
 	// Existing header values are overwritten if collision happens.
 	Headers map[string]string `mapstructure:"headers,omitempty"`
 
+	// Disable GZip compression.
```

Comment:
We should be consistent with gRPC config

Reply:
Fixed.